### ISSUE TITLE
 hax.psu top level site dark mode color issue

### ISSUE THREAD

**Issue by btopro** [haxtheweb/issues#2123]
![image](https://github.com/user-attachments/assets/5bc42f67-0b35-44c6-9c3f-7bbbedb93aa3)
this block is messed up if in darkmode at the platform level.

Something in this css block is offending

```css
details {
    overflow: hidden;
    display: flex;
    position: relative;
    max-width: 650px;
    padding: 0px;
    color: light-dark(black, rgb(255, 255, 255));
    margin: 0px;
}
```
